internal/database/security_event_logs.go

```diff
@@ -216,3 +249,24 @@ func (s *securityEventLogsStore) LogEventList(ctx context.Context, events []*Sec
 		}
 	}
 }
+
+func (s *securityEventLogsStore) LogSecurityEvent(ctx context.Context, eventName SecurityEventName, url string, userID uint32, anonymousUserID string, source string, arguments any) error {
+	argsJSON, err := json.Marshal(arguments)
+	if err != nil {
+		return errors.New("error marshalling arguments")
+
```

Comment:
```suggestion
		return errors.Wrap(err, "error marshalling arguments")
```

Might be worth seeing why the marshalling failed in the error